Kconfig

```diff
@@ -174,6 +174,12 @@ menu "LVGL configuration"
 			help
 				Align the start address of draw_buf addresses to this bytes.
 
+			config LV_DRAW_USE_GLOBAL_OPA_AND_MATRIX
```

Comment:
Will separate the config be better, considering some GPU could have no global alpha support?

Reply:
Separate configuration may cause the layer logic to become complicated. If the GPU does not support it, the `style_opa_layered` effect will not take effect. Users can be recommended to use `style_opa` to achieve similar effects.